app/components/insights-tabs.js

```diff
@@ -0,0 +1,23 @@
+import Component from '@ember/component';
+import { DEFAULT_INSIGHTS_INTERVAL, INSIGHTS_INTERVALS } from 'travis/services/insights';
+
+export const INSIGHTS_TABS = [
+  { slug: INSIGHTS_INTERVALS.WEEK, title: INSIGHTS_INTERVALS.WEEK.capitalize() },
+  { slug: INSIGHTS_INTERVALS.MONTH, title: INSIGHTS_INTERVALS.MONTH.capitalize() },
+];
+
+export default Component.extend({
+  tagName: 'ul',
+  classNames: ['travistab-nav travistab-nav--insights insights-tabs'],
+
+  selectedTab: DEFAULT_INSIGHTS_INTERVAL,
+  setTab: () => {},
```

Comment:
This could be just `setTab() {}`. Your variant works too, this one just a bit shorter 😄

Reply:
Nice, I like shorter :) Although, with the note below about binding to query params, I guess we may not even need this